.idea/.idea.osu/.idea/runConfigurations/RulesetTests__catch_.xml

```diff
@@ -0,0 +1,22 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="RulesetTests (catch)" type="DotNetProject" factoryName=".NET Project">
+    <option name="EXE_PATH" value="$PROJECT_DIR$/osu.Game.Rulesets.Catch.Tests/bin/Debug/netcoreapp2.0/osu.Game.Rulesets.Catch.Tests.dll" />
```

Comment:
Hmmm are you sure about this? All other projects (incl. framework) reference net471, except all ruleset configurations.

Reply:
i am not sure why i made them netcore.